forward/forward.go

```diff
@@ -124,7 +124,7 @@ func (f *Forwarder) ServeHTTP(w http.ResponseWriter, req *http.Request) {
 		return
 	}
 	log.Debugf("Round trip: %v, code: %v, duration: %v",
-		req.URL, response.StatusCode, time.Now().UTC().Sub(start))
+		reqClone.URL, response.StatusCode, time.Now().UTC().Sub(start))
```

Comment:
This is just a logging update so that we see the URL that's being sent rather than the one that was received.